core/eolearn/tests/test_utils/test_common.py

```diff
@@ -61,16 +61,10 @@ class ApplyToTestCase:
     function: Callable[[np.ndarray], np.ndarray]
     data: np.ndarray
     spatial_axes: Tuple[int, int]
-    expected: np.ndarray
+    expected: Optional[np.ndarray] = None
 
 
 APPLY_TO_TEST_CASES = [
-    ApplyToTestCase(
```

Comment:
I would change the name of the dataclass to `ApplyToAxesTestCase`, currently it looks like you're applying stuff to tests

Reply:
Agree changed.